### PR TITLE
Added regtest notes to backend README

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/1788

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -159,3 +159,58 @@ nodemon src/index.ts --ignore cache/ --ignore pools.json
...
+   ```
+   ./src/bitcoin-cli -named -regtest -rpcport=8332 sendtoaddress address=$(./src/bitcoin-cli -regtest -rpcport=8332 getnewaddress) amount=0.1 fee_rate=5
+   ```
+
+* See more example of `sendtoaddress`
+   ```
+   ./src/bitcoin-cli sendtoaddress # will print the help
+   ```
+
+* Mini script to generate txs with random fx rate (between 1 to 100 sat/vB). It's slow so don't expect to use this to test mempool spam, except if you let it run for a long time, or maybe with multiple regtest node connected to each other.
```
Comment (hunicus) [inline backend/README.md:202]: Haven't seen fx rate refer to feerate before...might be better to just say feerate?

**Review by hunicus — COMMENTED** [review]
This looks good to me overall.

Most of the changes I would suggest are minor wording and formatting stuff...I can suggest those in a separate PR.

File: backend/README.md (hunk 1 above)
Comment (wiz) [reply backend/README.md:202]: yeah FX rate is BTC/USD rate, must be a typo

Comment (wiz) [inline backend/README.md:202]:
```suggestion
* Mini script to generate transactions with random TX fee-rate (between 1 to 100 sat/vB). It's slow so don't expect to use this to test mempool spam, except if you let it run for a long time, or maybe with multiple regtest node connected to each other.
```

**Pull request merged**